tests/test_flask.py

```diff
@@ -61,7 +62,7 @@ def test_index(self):
         output = self.app.get('/')
         self.assertEqual(output.status_code, 200)
 
-        expected = """
+        expected = b"""
```

Comment:
Staying entirely in the binary domain rather than decoding makes sense to me for these cases.